README.md

```diff
@@ -412,6 +423,15 @@ scrollable will default to using the window for the scroll binding.
 
 You can optionally pass an offset value.   This value will be used when calculating if the bottom of the scrollable has been reached.
 
+* **eventDebounce**
+
+```hbs
+{{infinity-loader eventDebounce=50}}
+```
+
+Default is 50ms.  You can optionally pass a debounce time to delay loading the list when reach bottom of list
+>>>>>>> a2c68afb... Replace with ember-in-viewport (#242)
```

Comment:
This looks like a remnant from a merge conflict, otherwise let’s merge this thing!